priv/templates/nimble.phx.gen.template/test/support/factory.ex.eex

```diff
@@ -1,10 +1,5 @@
 defmodule <%= base_module %>.Factory do
   use ExMachina.Ecto, repo: <%= base_module %>.Repo
 
-  # Define your factories
-  # def user_factory do
-  #   %User{
-  #     email: sequence(:email, &"test-#{&1}@example.com")
-  #   }
-  # end
+  # Define your factories in /test/factories and declare it here, eg: `use <%= base_module %>.Accounts.UserFactory`
```

Comment:
```suggestion
  # Define factories in /test/factories and declare it here, 
  # eg: `use <%= base_module %>.Accounts.UserFactory`
```

Reply:
Changed in https://github.com/nimblehq/elixir-templates/pull/50/commits/7d8935c4be0ca9f4bac5e816ae80d0482352cc7d @hanam1ni 👍